utils/templates/ipamodule+member.py.in

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Authors:
+#   $author <$email>
+#
+# Copyright (C) $year Red Hat
+# see file 'COPYING' for use and warranty information
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.0",
+    "supported_by": "community",
+    "status": ["preview"],
+}
+
+DOCUMENTATION = """
+---
+module: ipa$name
+short description: Manage FreeIPA $name
+description: Manage FreeIPA $name and $name members
+options:
+  ipaadmin_principal:
+    description: The admin principal.
+    default: admin
+  ipaadmin_password:
+    description: The admin password.
+    required: false
+  name:
+    description: The list of $name name strings.
+    required: true
+    aliases: ["API_PARAMETER_NAME"]
+  PARAMETER1:
+    description: DESCRIPTION
+    required: REQUIRED
+    aliases: ["API_PARAMETER_NAME"]
+  PARAMETER2:
+    description: member DESCRIPTION
+    required: REQUIRED
+    aliases: ["API_PARAMETER_NAME"]
+  action:
+    description: Work on $name or member level.
+    choices: ["$name", "member"]
+    default: $name
+    required: false
+  state:
+    description: The state to ensure.
+    choices: ["present", "absent"]
+    default: present
+    required: true
+"""
+
+EXAMPLES = """
+# Ensure $name NAME is present
+- ipa$name:
+    name: NAME
```

Comment:
The `standard password` should be added here: `ipaadmin_password: SomeADMINpassword`.